packages/modules/json/device.py

```diff
@@ -17,9 +18,9 @@ def get_default_config() -> dict:
         "type": "json",
         "id": 0,
         "configuration": {
-            "ip_address": None
-            # ToDo: add protocol
-            # ToDo: add port
+            "protocol": "http",
+            "domain": None,
+            "port": ""
```

Comment:
Das ist eine numerische Angabe.
```suggestion
            "port": 80
```